docs/roadmap.rst

```diff
@@ -4,12 +4,16 @@ The tedana roadmap
 Project vision
 --------------
 
-ME-EPI processing is not well integrated into major preprocessing packages,
-yielding duplicated and unmaintained code.
-``tedana`` has been developed to address this need and will serve as a central repository
-for standard ME-EPI denoising as well as a testing ground for novel ME-EPI denoising methods.
-This will jointly reduce the external burden on pipeline maintainers,
-facilitate increased ME-EPI adoption, and enable future development in ME-EPI denoising.
+``tedana`` was originally developed as a place for the multi-echo fMRI
+denoising method that was originally defined in ME-ICA 
+(`ME-ICA source <https://github.com/ME-ICA/me-ica>`_. tedana was designed to
+be more understandable, modular, and adaptable so that it can serve as a
+testing ground for novel multi-echo fMRI denoising methods. We have expanded
+to welcome additional multi-echo fMRI processing approaches, and to support
+communal resources for multi-echo fMRI, whether or not people use the tedana
+software.
```

Comment:
Should we drop that ending part of the sentence? It sounds a bit weird to me. Maybe it's something a native English speaker would say, but I think we can drop that part because it doesn't add to the previous sentence.

Reply:
Which part, just after the last comma?

Reply:
Yes, that part. Sorry if that wasn't very clear.

Reply:
I tweaked the language a bit. I do want to emphasize that we welcome documentation additions and resources whether or not they are within what the tedana code is focused on.